pyinfra/facts/server.py

```diff
@@ -344,7 +344,7 @@ def process(output):
 
 class Crontab(FactBase):
     '''
-    Returns a dictionary of cron command -> execution time.
+    Returns a list of dictionaries where each entry corresponds to a single cron command.
```

Comment:
This is a _much_ cleaner approach - I always regretted the command -> details dict choice here! One problem here is that this is a breaking change - could we make this a new class (maybe just `Crons`? Or `CronEntries`?) and leave the old one as-is, I can then add a deprecation notice when used pending removal in v2.

Reply:
Ah, yeah, I can make that change. Will get to that when I have some more time to work on this PR.